<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>971</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>40</y>
        <width>181</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>10</y>
        <width>181</width>
        <height>571</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_open">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>370</y>
          <width>141</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>open</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_infor">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>219</width>
          <height>219</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>161</width>
           <height>201</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="horizontalSpacing">
           <number>19</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="1">
           <widget class="QLabel" name="label_filename">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_startdate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_type">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_duedate">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>filename</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>file type</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>start date</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>due date</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>751</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_exr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>691</width>
        <height>561</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>asset</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_asset">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>0</y>
          <width>701</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>clip</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_clip">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>691</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>exr</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_exr">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>671</width>
          <height>511</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>rig</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_rig">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>701</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>â˜…</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_bookmark">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>701</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>50</y>
        <width>251</width>
        <height>551</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTreeWidget" name="treeWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>241</width>
          <height>201</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="pushButton_reference">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>350</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>reference</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_import">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>300</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>import</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_infor_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>219</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>20</y>
           <width>221</width>
           <height>101</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <property name="horizontalSpacing">
           <number>19</number>
          </property>
          <property name="verticalSpacing">
           <number>8</number>
          </property>
          <item row="0" column="1">
           <widget class="QLabel" name="label_filename_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Saved time</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_savedtime">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>filename</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
